docker/Dockerfile

```
@@ -0,0 +1,81 @@
+FROM rocker/tidyverse:3.6.1
+MAINTAINER ccdl@alexslemonade.org
```

Because I keep seeing an warning in VScode that MAINTAINER is deprecated...
```suggestion
LABEL maintainer="ccdl@alexslemonade.org"
```